src/common/transformations/src/transformations/symbolic_transformations/symbolic_optimizations.cpp

```diff
@@ -210,10 +210,22 @@ bool ov::pass::SymbolicOptimizations::run_on_model(const std::shared_ptr<ov::Mod
     // it may break NNCF patterns and lead to unexpected FakeQuantize ops in the model.
     // So we decided to disable these passes in SymbolicOptimizations.
     const auto& pass_config = m_manager->get_pass_config();
+
+    const bool squeeze_was_enabled = !pass_config->is_disabled<EliminateSqueeze>();
+    const bool unsqueeze_was_enabled = !pass_config->is_disabled<EliminateUnsqueeze>();
+
     pass_config->disable<EliminateSqueeze>();
     pass_config->disable<EliminateUnsqueeze>();
 
     m_manager->run_passes(m);
+
+    if (squeeze_was_enabled) {
+        pass_config->enable<EliminateSqueeze>();
+    }
+    if (unsqueeze_was_enabled) {
+        pass_config->enable<EliminateUnsqueeze>();
+    }
```

Comment:
Could you please add a comment on why we have to do this (in order not to spoil the shared pass config)?